### PR TITLE
case switch using python

### PR DESCRIPTION
**Pull request by suyash2796**
### **Describe your change:**

* [x] Add an algorithm?
* [] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by suyash2796** [comment]
code review comments addressed

**Review by onlinejudge95 — CHANGES_REQUESTED** [review]
General formatting changes are required, please use the pre-commit hook for the same.

**Comment by suyash2796** [comment]
> General formatting changes are required, please use the pre-commit hook for the same.

Hi @onlinejudge95 -  code formatting is completed!

**Comment by onlinejudge95** [comment]
Also correct me if I am not wrong, Python 3.10 has a switch case construct build already?

**Comment by suyash2796** [comment]
> Also correct me if I am not wrong, Python 3.10 has a switch case construct build already?

oh yes! that was released yesterday :+1: . That was not available until 3.9 therefore as an alternative I implemented that for one of my projects and found it very useful so I thought that would be beneficial for the community. 
But I think that is not needed now as switch case problem is solved in 3.10!!
Thanks for your inputs. I really Appreciate it !

File: other/switch_case_with_python_function.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+def divide(attr1: int, attr2: int) -> int:
+    """
+    >>> divide(1,2)
+    0
+    """
+    return attr1 // attr2
+
+
+if __name__ == "__main__":
+    switch_case_mapper("multiply", 1, 2)
```
Comment (onlinejudge95) [inline other/switch_case_with_python_function.py:69]: Use doctests for testing

**Review by onlinejudge95 — CHANGES_REQUESTED** [review]

File: other/switch_case_with_python_function.py (hunk 1 above)
Comment (onlinejudge95) [reply other/switch_case_with_python_function.py:69]: Remove the following piece of code

**Review by onlinejudge95 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**